src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +133,30 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_sections="dk_bench",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_sections="dk_bench",
+)
+@click.option(
+    "--system-prompt",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_sections="dk_bench",
```

Comment:
@alimaredia We don't want `--system-prompt` to be specific to `dk_bench`. We can pass the system prompt to all of our evaluations.